Tetris/requirements.txt

```diff
@@ -0,0 +1,5 @@
+Package    Version
```

Comment:
just mention the package names with version like: pygame==20.2.3 without top heading and next dashed lines